pycyphal/transport/can/media/pythoncan/_pythoncan.py

```diff
@@ -189,6 +192,24 @@ def __init__(
         self._maybe_thread: typing.Optional[threading.Thread] = None
         self._rx_handler: typing.Optional[Media.ReceivedFramesHandler] = None
         self._background_executor = concurrent.futures.ThreadPoolExecutor(max_workers=1)
+        # This is for communication with a thread that handles the call to _bus.send
+        self._tx_queue: queue.Queue = queue.Queue()
+
+        def transmit_thread_worker(
```

Comment:
Please make this a method and remove the queue parameter because it is reachable via `self`.